lib/capistrano/tasks/deploy.rake

```diff
@@ -148,8 +148,10 @@ namespace :deploy do
   desc "Clean up old releases"
   task :cleanup do
     on release_roles :all do |host|
-      releases = capture(:ls, "-xtr", releases_path).split
-      if releases.count >= fetch(:keep_releases)
+      releases = capture(:ls, "-x", releases_path).split
```

Comment:
Since you have removed the `-r` flag, doesn't that mean the releases are no longer in reverse order? If so, the Ruby code for choosing which releases to delete needs to be flipped, right?